admin/pre_reg/serializers.py

```diff
@@ -33,7 +33,7 @@ def serialize_draft_registration(draft, json_safe=True):
         'registration_schema': serialize_meta_schema(draft.registration_schema),
         'initiated': iso8601format(draft.datetime_initiated) if json_safe else draft.datetime_initiated,
         'updated': iso8601format(draft.datetime_updated) if json_safe else draft.datetime_updated,
-        'submitted': iso8601format(draft.approval.initiation_date) if json_safe else draft.approval.initiation_date,
+        'submitted': iso8601format(draft.approval.initiation_date) if json_safe else draft.approval.initiation_date if draft.approval is not None else None,
```

Comment:
I don't know when json_safe is True, but if it is ever `True` and there is no `draft.approval`, this will fail. 

It reads in order:
**1** if True else 2 if False else 3 == 1
**1** if True else 2 if True else 3 == 1
1 if False else **2** if True else 3 == 2
1 if False else 2 if False else **3** == 3